openverse_catalog/templates/create_provider_ingester.py

```diff
@@ -0,0 +1,180 @@
+"""
+Script used to generate a templated ProviderDataIngester.
+"""
+
+import argparse
+import re
+from pathlib import Path
+
+import inflection
+
+
+TEMPLATES_PATH = Path(__file__).parent
+REPO_PATH = TEMPLATES_PATH.parent
+PROJECT_PATH = REPO_PATH.parent
+MEDIA_TYPES = ["audio", "image"]
+
+
+def _render_provider_configuration(provider: str, media_type: str):
+    """
+    Render the provider configuration string for a particular media type.
+    """
+    return f'"{media_type}": prov.{provider}_{media_type.upper()}_PROVIDER,'
+
+
+def _get_filled_template(
+    template_path: Path, provider: str, endpoint: str, media_types: list[str]
+):
+    with template_path.open("r", encoding="utf-8") as template:
+        camel_provider = inflection.camelize(provider)
+        screaming_snake_provider = inflection.underscore(provider).upper()
+
+        # Build provider configuration
+        provider_configuration = "\n        ".join(
+            _render_provider_configuration(screaming_snake_provider, media_type)
+            for media_type in media_types
+        )
+
+        template_string = template.read()
+        script_string = (
+            template_string.replace("{provider}", camel_provider)
+            .replace("{screaming_snake_provider}", screaming_snake_provider)
+            .replace("{provider_underscore}", inflection.underscore(provider))
+            .replace("{provider_data_ingester}", f"{camel_provider}DataIngester")
+            .replace("{endpoint}", endpoint)
+            .replace("{provider_configuration}", provider_configuration)
+        )
+
+        return script_string
+
+
+def _render_file(
+    target: Path,
+    template_path: Path,
+    provider: str,
+    endpoint: str,
+    media_types: list[str],
+    name: str,
+):
+    with target.open("w", encoding="utf-8") as target_file:
+        filled_template = _get_filled_template(
+            template_path, provider, endpoint, media_types
+        )
+        target_file.write(filled_template)
+        print(f"{name + ':':<18} {target.relative_to(PROJECT_PATH)}")
+
+
+def fill_template(provider, endpoint, media_types):
+    print(f"Creating files in {REPO_PATH}")
+
+    dags_path = TEMPLATES_PATH.parent / "dags" / "providers"
```

Comment:
Can we use this here?
```suggestion
    dags_path = PROJECT_PATH / "dags" / "providers"
```

Reply:
No, currently it would need to be REPO_PATH. But I think the use of PROJECT/REPO is wrong, I will update that.